core/src/components/chip/chip.scss

```diff
@@ -19,13 +19,12 @@
 
   align-items: center;
 
-  height: 32px;
+  min-height: calc(32 / 14 * 1em);
```

Comment:
What happens if the `32px` is left in?

Reply:
If we leave a min-height of 32px, then of course chips at small font sizes are now large. I suppose this might actually be what we want, but the ticket has a requirement of dynamic heights, so I assumed that meant in both directions. (But perhaps not.)

Reply:
Another option is that we could remove the height & min-height entirely and they'll just scale on their own. I think the only chips that look weird in this situation are chips that only have an icon, no label. I'm not sure if those are even really supported.

Reply:
Good point, why not use `2rem`? It would scale properly while keeping the default of 32px.

Reply:
> but the ticket has a requirement of dynamic heights, so I assumed that meant in both directions. (But perhaps not.)

"dynamic height" means that the height can adjust according to the size of the text. In this case, `min-height: 32px` is what we want. This will retain the correct dimensions with the default text size but still allow the chip to scale up if the font size increases.